<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><!-- InstanceBegin template="/Templates/index.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<!-- InstanceBeginEditable name="doctitle" -->
<title>He Group OntoDen </title>
<!-- InstanceEndEditable --><meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link rel="shortcut icon" href="/favicon.ico" />
<link href="../styleMain.css" rel="stylesheet" type="text/css">
<!-- InstanceBeginEditable name="head" -->
<style type="text/css">
<!--
.smallContent .footerlink em {
	color: #800000;
}
-->
</style>
<!-- InstanceEndEditable -->
<style type="text/css">
<!--
.style3 {font-size: 14px}
-->
</style>
</head>

<body>
<table align="center" cellpadding="0" cellspacing="0" width="900px" style="border:1px solid #999966">
  <tr>
    <td><table bgcolor="#ffcc33" border="0" cellpadding="0" cellspacing="0" width="900px">
        <tr class="content">
          <td><img src="../images/pixel.gif" height="5" width="1"></td>
        </tr>
      </table>
      <table bgcolor="#003366" border="0" cellpadding="5" cellspacing="0" width="900px">
        <tr>
          <td width="130" height="90" align="center" valign="middle">&nbsp;</td>
          <td width="420" align="center" valign="middle" class="emhead"><a href="/index.php"><img src="../images/HeGroup.gif" alt="He Group" width="188" border="0"></a></td>
          <td align="center" valign="middle" width="140">&nbsp;</td>
        </tr>
      </table>
      <table bgcolor="#003366" border="0" cellpadding="3" cellspacing="0" width="900px">
        <form name='webSearch' action='../searchWeb.php' method='POST' target="_blank">
        <tr>
          <td align="center" valign="middle" bgcolor="#ffcc33">
            <a href="../index.php" class="topnav">Home</a><span class="temphead">|</span><a href="../aboutUs/aboutUs.html" class="topnav">People</a><span class="temphead">|</span><a href="../projects.html" class="topnav">Projects</a><span class="temphead">|</span><a href="../publications.html" class="topnav">Publications</a><span class="temphead">|</span><a href="../faqs.php" class="topnav">FAQs</a><span class="temphead">|</span><a href="../links.html" class="topnav">Links</a><span class="temphead">|</span><a href="../news.php" class="topnav">News</a><span class="temphead">|</span><a href="../contacts/index.php" class="topnav">Contact Us</a></td>
        </tr>
        </form>
      </table>

      <table width="900px" border="0" cellpadding="5" cellspacing="0" bgcolor="#ffffcc">
        <tr>
          <td valign="top" style="height:400px;"><!-- InstanceBeginEditable name="Main" -->
           <p align="center" class="mainhead">Ontozoo           </p>
           <!--table width="725" border="1">
             <tr>
               <td width="715"><p class="smallContent"><strong class="footerlink"><em>Annoucement: </em></strong> An <a href="http://kr-med.org/icbofois2012/icbo.htm">ICBO 2012</a> workshop entitled <a href="http://kr-med.org/icbofois2012/vdosme/index.htm">&quot;Vaccine and Drug Ontology in the Study of Mechanism and Effect&quot; (VDOSME 2012)</a> will be hold in the end of July, 2012, in the Medical University of Graz, Austria.  VDOSME 2012 will provide a platform for discussing problems and solutions in the   development and application of biomedical ontologies to representing and   analyzing vaccines/drugs, vaccine/drug administrations,   vaccine/drug-induced immune responses, and similar topics. </p>                </td>
              </tr>
           </table-->
           <table width="860" align="center">
              <tr>
                <td>
           <p>Ontozoo is a web page that describes all the ontology software programs developed in the He Group. </p>
           <p>Onto-animal  tools are a package of web-based ontology tools developed to support efficient  and integrated ontology development and application. This package of tools  includes OntoFox, Ontodog, Ontorat, Ontobee, Ontobeep,  Ontobat, and GOfox. Each tool has  specific functions; together, these tools support the extraction of a single or  subset of terms and community views from existing ontologies, generation and  editing of ontology terms, query and visualization of ontology terms,  comparison among ontologies, and instance-level data representation and  analysis. Based on the Web Ontology Language (OWL) and Semantics Web  technologies, these tools have widely been used by thousands of ontology  developers in over 20 communities. </p>
           <p><strong>Table of contents</strong>:</p>
           <ol  type="A">
             <li><a href="#software">Introduction of Onto-animal tools </a></li>
             <li><a href="#ontofox">OntoFox </a></li>
             <li><a href="#ontobee">Ontobee </a></li>
             <li><a href="#ontobeep">Ontobeep</a></li>
             <li><a href="#ontodog">Ontodog</a></li>
             <li><a href="#ontodog">Ontorat</a></li>
             <li><a href="#ontobat">Ontobat</a></li>
             <li><a href="#gofox">GOfox</a></li>
             <li><a href="#bfoconvert">BFOConvert</a></li>
             <li><a href="#hegrouptriplestore">Hegroup RDF triple store</a></li>
             <li><a href="#acknowledgements">Acknowledgements</a></li>
           </ol>
           <p id="software"><span class="mainhead">A. Introduction of Onto-animal tools:   </span></p>
<p>Biological/biomedical ontologies are sets of  computer- and human-interpretable terms and relations that represent entities  and their relations in the biological/biomedical world. Biomedical ontologies  have emerged as a major tool for the integration and analysis of the large  amounts of heterogeneous biological data available in the post-genomics era. </p>
<p>To support ontology development and applications, we have  developed a collection of &ldquo;Onto-animal&rdquo; tools, including OntoFox,  Ontodog, Ontorat, Ontobee,  Ontobeep, and Ontobat. The back-end of these Onto-animal tools is the  He group&rsquo;s RDF triple store,  which has become the default ontology RDF triple store for the Open Biological  and Biomedical Ontologies (OBO) Foundry ontologies.  </p>
<p> The Onto-animal tools are summarized in the following figure and described in the following sections: </p>
<p align="center">
<img src="../ontoden/images/HegroupOntologyTools.jpg" width="697" height="187" >
  <br/><br/>
</p>
<p align="left"><strong>Reference:</strong> He Y, Zheng J, Lin Y. Onto-animal tools for reusing ontologies, generating and editing ontology terms, and dereferencing ontology terms.<em><em>International Conference on Biomedical Ontologies (ICBO</em>-2015).&nbsp;</em>Lisbon, Portugal, July 27-30, 2015. 2-page short paper for poster presentation and flask talk. [<a href="docs/Ontoanimal_poster_ICBO2015_FlashTalk.pdf">Flash Talk</a>] [<a href="docs/Ontoanimal_poster_ICBO2015.pdf">Poster</a>] [<a href="docs/Onto-animal_ICBO_2015_paper.pdf">Short paper</a>].</p>

<p align="left" id="ontofox"><span class="mainhead">B. OntoFox: Extract ontology terms and axioms</span> (<a href="http://ontofox.hegroup.org/">http://ontofox.hegroup.org/</a>)</p>
<p align="left"><strong> OntoFox:</strong> An <span class="style3"><strong>Onto</strong></span>logy tool that <span class="style3"><strong>F</strong></span>etches <span class="style3"><strong>o</strong></span>ntology terms and a<span class="style3"><strong>x</strong></span>ioms. OntoFox is a web-based system to support ontology reuse. </p>
<p>Reusing a portion of an existing  ontology is often required in the ontology development process. After a user  provides a single or a set of terms of interest, OntoFox  is able to  fetch selected classes, properties, annotations, and their related terms from  source ontologies and save the results in the OWL format (Xiang et al., 2010). OntoFox implements the <a href="http://obi-ontology.org/page/MIREOT">Minimum Information to  Reference an External Ontology Term (MIREOT)</a> strategy by extracting minimum information  of requested terms.  In addition, by providing different options, OntoFox can extract different  levels of intermediate terms between the required terms and a chosen higher  level or top term. Inspired by existing <a href="https://en.wikipedia.org/wiki/Ontology_modularization">ontology modularization</a> techniques. OntoFox also implements a  new SPARQL-based ontology term extraction algorithm that extracts all terms and  axioms related to a given set of user-provided terms. 
<p align="left"><strong>Reference:</strong> Xiang Z, Courtot M, Brinkman RR, Ruttenberg A, He Y. <a href="http://www.biomedcentral.com/1756-0500/3/175">OntoFox: web-based support for ontology reuse</a>. <em>BMC Research Notes</em>. 2010, <strong>3:</strong>175. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20569493">20569493</a>] </p>

<p align="left" id="ontobee"><span class="mainhead">C. Ontobee: Linked data server for web displaying and dereferencing ontology terms </span>(<a href="http://www.ontobee.org">http://www.ontobee.org</a>)</p>
<p>Ontobee   is an ontology browser and a linked ontology data server for dereferencing ontology  terms.  To date, there are &gt;100 ontologies listed on Ontobee. Ontobee loads individual  page for each term in an ontology. All related information of a single term,  such as label, definition, synonyms, superclass hierarchy, logical axioms, and  term usage by other ontologies is displayed. In addition, for each ontology, Ontobee  generates statistics with counts of classes, object properties, annotation  properties, and datatype properties based on term ontology prefixes. Furthermore,  Ontobee automatically provides an Excel document listing all terms in an  ontology. Ontobee provides ontology term search and SPARQL query service  supported by the He group triple store. Ontobee is a<em> de facto</em> search engine for OBO Foundry ontologies.</p>
<p align="left"><strong>Reference:</strong> Xiang Z, Mungall C, Ruttenberg A, He Y. <a href="http://www.ontobee.org/Ontobee_ICBO-2011_Proceeding.pdf">Ontobee: A Linked Data Server and Browser for Ontology Terms</a>. <em>Proceedings of the 2nd International Conference on Biomedical Ontologies (ICBO)</em>, July 28-30, 2011, Buffalo, NY, USA. Pages 279-281. URL: <a href="http://ceur-ws.org/Vol-833/paper48.pdf">http://ceur-ws.org/Vol-833/paper48.pdf</a>. </p>

<p align="left" id="ontobeep"><span class="mainhead">D. Ontobeep: Ontology comparison  </span> (<a href="http://www.ontobee.org/ontobeep/">http://www.ontobee.org/ontobeep/</a>)</p>
<p>Ontobeep   is an ontology comparison program. Ontobeep can be used to compare different  ontologies by aligning them from the roots of these ontologies. The alignment  identifies common terms existing in two or three ontologies. Ontobeep also provides  a statistic report of the alignment analysis. Ontobeep may be utilized to  detect inconsistency and term duplication in one or more ontologies. </p>
<p align="left"><strong>Reference:</strong></p>

<p align="left" id="ontodog"><span class="mainhead">E. Ontodog: Generate ontology community view </span>(<a href="http://ontodog.hegroup.org/">http://ontodog.hegroup.org/</a>)</p>
<p>Similar to OntoFox, the  web-based Ontodog program   is able to extract a subset of ontology terms and axioms. Unlike OntoFox,  Ontodog includes two unique features. First, Ontodog allows the generation of  an ontology community view, which we have defined as &ldquo;the whole or a subset of  the source ontology with <em>user-specified  annotations including user preferred labels</em>&rdquo;. Second, Ontodog uses  Excel input files to identify which terms to retrieve and to add user-specified  annotations. Excel templates are also provided for easy implementation.&nbsp; </p>
<p align="left"><strong>Reference: </strong>Zheng J, Xiang Z, Stoeckert   Jr. CJ, He Y. <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btu008?ijkey=zrGC7jDtBowtltp&amp;keytype=ref">Ontodog: a web-based ontology community view generation   tool</a>. <em>Bioinformatics</em>. 2014; doi:  10.1093/bioinformatics/btu008. [<a href="http://bioinformatics.oxfordjournals.org/cgi/reprint/btu008?ijkey=zrGC7jDtBowtltp&amp;keytype=ref">PDF</a>] PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24413522">24413522</a>.</p>

<p align="left" id="ontorat"><span class="mainhead">F. Ontorat: Adding new terms and new axioms to an ontology  based on design pattern&nbsp;</span>(<a href="http://ontorat.hegroup.org">http://ontorat.hegroup.org</a>)</p>
<p><strong>Ontorat:</strong> An ontology tool that supports <span class="style3"><strong>Onto</strong></span>logy <span class="style3"><strong>r</strong></span>epresentation of <span class="style3"><strong>a</strong></span>xioms using <span class="style3"><strong>t</strong></span>emplates and design patterns. </p>
<p>The Ontorat program  automatically  generates and edits ontology terms and axioms and provides term annotations. Ontorat uses reusable ontology design patterns  (ODPs) to solve recurrent modeling problems. A specific ODP can be used to derive  an Excel template of different terms/annotations and a set of rules that define  the relations among those terms/annotations. An Ontorat template is similar to  a <a href="http://obi-ontology.org/page/Quick_Term_Templates">QTT (Quick Term Template)</a>. Such a template can be  populated with specific terms or annotations to define or annotate specific  ontology terms. With the support of the Ontorat settings, the populated  template spreadsheet can then be converted into an OWL file with newly  generated ontology terms and axioms.&nbsp; </p>
<p align="left"><strong>Reference:</strong> Xiang Z, Zheng J, Lin Y, He Y.&nbsp;<a href="http://www.jbiomedsem.com/content/6/1/4/abstract">Ontorat: Automatic generation of new ontology terms, annotations, and axioms based on ontology design patterns</a>.&nbsp;<em>Journal of Biomedical Semantics</em>. 2015,&nbsp;<strong>6</strong>:4&nbsp; doi:10.1186/2041-1480-6-4. PMID:&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/25785185">25785185</a>.</p>

<p align="left" id="ontobat"><span class="mainhead">G. Ontobat: Ontology-based data analysis&nbsp;</span>(<a href="http://ontobat.hegroup.org">http://ontobat.hegroup.org</a>)</p>
<p align="left"><strong>Ontobat: </strong>An <span class="style3"><strong>Onto</strong></span>logy-based <span class="style3"><strong>b</strong></span>iodata <span class="style3"><strong>a</strong></span>nalysis <span class="style3"><strong>t</strong></span>ool.</p>
<p>Ontobat is  a web-based Semantic Web tool to support ontology-based biological  data query and analysis. As a use case, Ontobat includes a program called OntoCOG:, which is an ontology-based statistical tool for  analysing the Clusters of Orthologous Groups  of proteins (<a href="http://www.ncbi.nlm.nih.gov/COG/" class="style3"><strong>COG</strong></a>). </p>
<p>Unlike other Onto-animal tools, Ontobat  focuses on  instance level ontology data generation and analysis&nbsp; (Xiang et al., 2014). Ontobat aims to  support Linked Open Data (LOD) generation, upload, query, browsing, and  statistical analysis. Many features of Ontobat are still under development. </p>
<p align="left"><strong>Reference:</strong> </p>

<p align="left" id="gofox"><span class="mainhead">H. GOfox: </span>(<a href="http://gofox.hegroup.org">http://gofox.hegroup.org</a>)</p>
<p align="left">&nbsp;</p>
<p align="left"><strong>Reference:</strong></p>
<p align="left">&nbsp;</p>
<p align="left" id="bfoconvert"><span class="mainhead">I. BFOConvert: </span>(<a href="http://bfoconvert.hegroup.org">http://bfoconvert.hegroup.org</a>)</p>
<p align="left">&nbsp;</p>
<p align="left"><strong>Reference:</strong></p>
<p align="left"></p>
<p align="left" id="hegrouptriplestore"><span class="mainhead">J. Hegroup RDF triple store </span><a href="http://sparql.hegroup.org/sparql/">(http://sparql.hegroup.org/sparql/)</a></p>
<p align="left">Most of our ontology tools use our Hegroup <a href="http://en.wikipedia.org/wiki/Resource_Description_Framework" title="Resource Description Framework">Resource Description Framework</a> (RDF) triple store. The  SPARQL endpoint of our triple store ... </p>
<p align="left">&nbsp;</p>
<p><strong>In summary, </strong>the web-based Onto-animal tool package provides a set of  comprehensive tools to support ontology development and applications. These  tools save time and efforts for ontology developers and users, especially those  who do not have or have limited software programming background.&nbsp;&nbsp; </p>
<p align="left" id="acknowledgements"><span class="mainhead">K. Acknowledgements:  </span></p>
<p align="left">The  Onto-animal tools were primarily developed in He group at the University of Michigan. The major software developers include Zuoshuang (Allen) Xiang, Bin Zhao, Edison Ong, and Oliver He. Asiyah Yu Lin, a previous postdoc research fellow in He group, contributed a lot by testing the tools with use cases and providing feedbacks. All the source code of these tools were generated in He group. This research was supported by a NIH R01 grant  (1R01AI081062). </p>
<p align="left">The development of Onto-animal tools has also obtained supports and collaborations from many collaborators outside the University of Michigan, including Melanie Courtot, Alan Ruttenberg, Chris Mungall, Jie Zheng, among  others.  The major collaborators of each Onto-animal tools are also co-authors of the Ontoanimal papers. </p>
<p align="left">&nbsp;</p>
<p align="left">Your suggestions and comments are welcome. You can directly contact <a href="../aboutUs/Oliver.html">Oliver He</a> (Ontozoo keeper) and send a message through our<a href="../feedback/index.php"> feedback website</a>. Thank you. </p>
<p align="left"><strong><em>Note:</em></strong> This page is still under construction.</p>
                </td>
              </tr>
            </table>
          <!-- InstanceEndEditable -->          </td>
        </tr>
      </table>
      <table bgcolor="#ffcc33" border="0" cellpadding="0" cellspacing="0" width="900px">
        <tr>
          <td height="25" align="center"><a href="http://sitemaker.umich.edu/helab" class="footerlink">He Lab </a><span class="subhead">|</span><a href="http://www.ontobee.org/" class="footerlink">Ontobee</a><span class="subhead">|</span><a href="http://ontofox.hegroup.org" class="footerlink">OntoFox</a><span class="subhead">|</span><a href="http://www.oae-ontology.org" class="footerlink">OAE</a><span class="subhead">|</span><a href="http://www.violinet.org/vaccineontology" class="footerlink">VO</a><span class="subhead">|</span><a href="http://genomesh.hegroup.org" class="footerlink">Genomesh</a><span class="subhead">|</span><a href="http://ignet.hegroup.org" class="footerlink">Ignet</a><span class="subhead">|</span><a href="http://www.minituba.org" class="footerlink">miniTUBA</a><span class="subhead">|</span><a href="http://www.phidias.us" class="footerlink">PHIDIAS</a><span class="subhead">|</span><a href="http://www.violinet.org" class="footerlink">VIOLIN</a></td>
        </tr>
      </table>
    </td>
  </tr>
  <tr>
    <td><table border="0" cellpadding="8" cellspacing="0" width="100%">
        <tr>
          <td valign="top" bgcolor="#ffffcc" style="padding-left:22px;">He Group <br>
          University of Michigan Medical School<br> Ann Arbor, MI 48109</td>
          <td width="175" align="center" valign="middle" bgcolor="#ffffcc" class="smallcontent"><a href="http://www.umich.edu" target="_blank"><img src="../images/wordmark_m_web.jpg" alt="U of M" width="166" height="20" border="0"></a></td>
		  <td width="100" align="center" valign="middle" bgcolor="#ffffcc" class="smallcontent" style="padding-right:22px;"><a href="http://www.umich.edu" target="_blank"></a><a href="http://www3.niaid.nih.gov/"><img src="../images/niaid.gif" alt="NIAID" width="45" height="45" border="0"></a><span class="style3"><a href="http://www3.niaid.nih.gov/">NIAID</a></span></td>
        </tr>
      </table></td>
  </tr>
</table>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-4869243-2");
pageTracker._initData();
pageTracker._trackPageview();
</script>
</body>
<!-- InstanceEnd --></html>
